### ISSUE TITLE
DE on log scale gene expression values gives spurious results

### ISSUE THREAD

**Issue by sidneymbell** [chanzuckerberg/cellxgene#103]
Josh Batson's done some nice work showing that a ttest for differential expression on log scale values gives super inaccurate results. This is far less of a problem when using linear scale values. 

If we still intend to intercept the user at the point where their expression matrix is fully normalized, etc., then perhaps we need a flag indicating the scale of those values.

**Comment by freeman-lab** [comment]
Agreed this is a concern, the tricky thing is that the typical dataset lacks information about _how_ it was preprocessed, so our only clue to whether a given dataset is log normalized or not is some calculation about the data's scale or other statistics, which would add non-trivial computation to our launch process.

BTW, our `prepare` method does not log transform, unless you explicitly specify a `recipe` that does, which seems right.

I think that's the best we can do, and should close this one.

**Comment by freeman-lab** [comment]
Also: I'm hoping scanpy's differential expression methods will be smart(ish) about this, which will mitigate the issue once we start using those.
